client/components/home/TemplatesGrid.tsx

```
@@ -0,0 +1,158 @@
+// Extract entity ID from query label (everything after the last dash)
+function extractEntityId(label: string): string | null {
+  const parts = label.split(" - ");
+  if (parts.length > 1) {
+    return parts[parts.length - 1];
+  }
+  return null;
+}
+
+// Extract category emoji and name from label
+function extractCategory(label: string): { emoji: string; name: string } {
+  const emojiMatch = label.match(/^([🎯🏗️🏃‍♂️🔍⚡🚀]+)\s+(.+)/);
```

Biome (2.1.2)

[error] A character class cannot match a surrogate pair. Add the 'u' unicode flag to match against them.

A surrogate pair forms a single codepoint, but is encoded as a pair of two characters. Without the unicode flag, the regex matches a single surrogate character.
Safe fix: Add unicode u flag to regex


(lint/suspicious/noMisleadingCharacterClass)